src/applications/disability-benefits/all-claims/components/IndividualsInvolvedCard.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function IndividualsInvolvedCard({ formData }) {
+  const { name } = formData;
+  const serviceMemberTitle = formData['view:serviceMember']
```

Comment:
I find these ternaries kind of hard to read. What about:

```js
let displayTitle;

if (name.first || name.last) {
  displayTitle = `${name.first || ''} ${name.last || ''}`
} else {
  displayTitle = formData['view:serviceMember'] 
    ? 'Servicemember'
    : 'Civilian';
}
```

Reply:
Sure I'm all for readability. The Linter with line length does effect lengthy turneries!